.github/workflows/run-test.yml

```diff
@@ -24,7 +24,7 @@ jobs:
 
     services:
       rabbitmq:
-        image: rabbitmq:3-alpine
+        image: rabbitmq:4-alpine
```

Comment:
Not sure if this was appropriate to bump, but bumping it without the fix in this PR demonstrates the problem.